torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -49,21 +49,36 @@ kir::Bool* UnrollPass::getThreadPredicate(const kir::TensorView* tv) {
 }
 
 void UnrollPass::handle(kir::Expr* expr) {
-  // If tv op, predicate it (except for top level expressions)
-  if (ir_utils::isTVOp(expr) && !for_loops_.empty()) {
+  if (ir_utils::isTVOp(expr)) {
+    // If tv op, predicate it (except for top level expressions)
```

Comment:
Please remove "except for top level expressions"

Reply:
Updated Thanks!